keyboards/ergodone/keymaps/vega/keymap.c

```diff
@@ -0,0 +1,606 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define BASE 0 // default layer, Dvoark
```

Comment:
Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```c
enum layer_names {
    BASE,
    GREL,
    GREU,
    SYMB,
    MATH,
    QWER,
    FNLR
};
```